### PR TITLE
changes to contributing.md

### LINKED ISSUE #49: Update "Contributing" doc
Add to contributing: 
* [x] " check to make sure foreman is set to run at port 5000. If it is not, you will not be able to be authenticated with MyUSA. "
* [x] In authentication section, link to https://alpha.my.usa.gov/applications to make sure you are on the list of allowed local developers

### PR DESCRIPTION
**Pull request by gemfarmer**
Fixes #49

### CODE REVIEW

**Comment by gemfarmer** [comment]
@jessieay @ccostino let me know if I should add anything!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ is [purposefully excluded from the project's `Gemfile`][exclude].
 [foreman]: https://github.com/ddollar/foreman
 [exclude]: https://github.com/ddollar/foreman/pull/437#issuecomment-41110407
 
-The application will run locally at `http://localhost:5000/`
+The application will run locally at `http://localhost:5000/`. 
+
+If you have previously run a project on a different port, a `.foreman` file may be generated at the root of your directory. If so, make sure that this file is set to port `5000` or you will be unable to authenticate locally with MyUSA.
```
Comment (jessieay) [inline CONTRIBUTING.md]: you OK with doing line breaks at around 80 chars? makes it a lot easier to leave inline comments in github and navigate code/words when working on a project

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -43,4 +45,6 @@ you can contact Jessie Young. Otherwise, you can create a Slack bot
 ### Authentication
 
 You will need to be on the developer list to authenticate locally via MyUSA.
-Contact Brian Hedberg to be added to the developer list.
+Contact Brian Hedberg to be added to the developer list. If you are on the list, `dolores-local` will be one of your [Authorized Applications](https://alpha.my.usa.gov/authorizations) on MyUSA.
```
Comment (jessieay) [inline CONTRIBUTING.md]: maybe we should suggest *how* you should be contacted?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,4 +45,6 @@ you can contact Jessie Young. Otherwise, you can create a Slack bot
...
+
+If `dolores-local` is on your MyUSA list for Authorized Applications and you are still unable to authenticate, check with Brian to make sure that the `MYUSA_KEY` and `MYUSA_SECRET` keys listed in `.env` are up to date.
```
Comment (jessieay) [inline CONTRIBUTING.md]: are we suggesting where people get these keys in the first place?

**Comment by jessieay** [comment]
Left a few small comments, feel free to address and merge and your leisure :umbrella:

File: CONTRIBUTING.md (hunk 1 above)
Comment (gemfarmer) [inline CONTRIBUTING.md]: Change made. I will keep this in mind for future PRs

File: CONTRIBUTING.md (hunk 2 above)
Comment (gemfarmer) [inline CONTRIBUTING.md]: I will just make a note about being internal to 18F.

File: CONTRIBUTING.md (hunk 3 above)
Comment (gemfarmer) [inline CONTRIBUTING.md]: We do in the Required Keys section.

**Pull request merged**